docs/about/FAQ.md

```diff
@@ -46,8 +55,17 @@ list.
 
 ## Is there an online service?
 
-Currently there isn't an online service, but having one is part of our
-priorities for the v1.
+Yes! It's in preview so there might be some intermitent issues, but
+you can scan an online website in [here][scanner].
+
+We are working on tweaking the list of rules we run and its
+configuration, but if it is a recommended one it will probably be run.
+Also, it uses the `jsdom connector` so there might be unexpected
+results.
```

Comment:
Maybe use:

> We are working on tweaking the list of rules it runs and its
configuration, but if a rule is a recommended one it will probably 
be run. 
> 
> Also, note that the scanner uses the `jsdom connector` so there 
might  be unexpected results.